playground/assets/settings.scss

```diff
@@ -1,5 +1,5 @@
 /* DON'T USE @use here */
-@forward 'vuetify/settings' with (
-  $utilities: false,
-  $button-height: 40px,
+@use 'vuetify/settings' with (
```

Comment:
can you add another sass file keeping the original? this was to override same variables (it is used in combination with the configFile)

Reply:
or rename the original to another name (maybe renaming also main.scss)

Reply:
I'm sorry, I sent an adjustment

What do you think of custom-vuetify.scss?

Reply:
![image](https://github.com/userquin/vuetify-nuxt-module/assets/43552865/dd0f70f6-0c17-4c89-acae-d7d140604e05)